data_dir: "./DataCollected/prismNewVP/"
model_type: "PCA_LDA" #[PCA | PCA_LDA], null and heuristic coming soon
PCALDA_component_number: 26
PCA_component_number: 7